bin/cron/schedule.js

```diff
@@ -1,216 +1,16 @@
-
-            cron[0] = frequencyValue === 60
-                ?   curr
-                :   mod > 0
-                    ?  (curr % frequencyValue) + '-' + (59) + '/' + (frequencyValue)
-                    :   '*/' + (frequencyValue);
-
-            return cron.join(' ');
-        },
-    },
-    hours: {
-        abbrev: ['h', 'hour', 'hours' ],
-        values: [ 1, 2, 3, 4, 6, 8, 12, 24],
-        encode(frequencyValue) {
-            const now = new Date();
-            const cron = [now.getMinutes(), '*', '*', '*', '*'];
-            const curr = now.getHours();
-            const mod = curr % frequencyValue;
-
-            cron[1] = frequencyValue === 24
-                ?   curr
-                :   mod > 0
-                    ?  (curr % frequencyValue) + '-' + (23) + '/' + (frequencyValue)
-                    :   '*/' + (frequencyValue);
-
-            return cron.join(' ');
-        },
-    },
-    days: {
-        abbrev: ['d', 'day', 'days'],
-        values: [ 1 ],
-        encode() {
-            return intervals.hours.encode(24);
-        },
-    },
-};
-
-
-const createCommand = require('../create');
-
 
 module.exports = Cli.createCommand('schedule', {
-    description: 'Schedule a webtask to run periodically',
-    plugins: [
-        require('../_plugins/profile'),
-    ],
-    optionGroups: _.extend({}, createCommand.optionGroups, {
-        'Cron options': {
-            'no-auth': {
-                description: 'Disable cron webtask authentication',
-                dest: 'noAuth',
-                type: 'boolean',
-            },
-            'state': {
-                description: 'Set the cron job\'s state',
-                choices: ['active', 'inactive'],
-                defaultValue: 'active',
-                type: 'string',
-            },
-            'tz': {
-                description: `An IANA timezone name (see: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), relative to which the cron schedule will be calculated. If not specified, the webtask cluster's default timezone is used.`,
-                metavar: 'TIMEZONE',
-                type: 'string',
-            },
-        },
-    }),
-    options: _.extend({}, createCommand.options, {
-
-    }),
-    params: _.extend({}, {
-        'schedule': {
-            description: 'Either a cron-formatted schedule (see: http://crontab.guru/) or an interval of hours ("h") or minutes ("m"). Note that not all intervals are possible.',
-            type: 'string',
-            required: true,
-        },
-    }, createCommand.params),
-    epilog: [
-        'Examples:',
-        '1. Create a webtask that runs every five minutes:',
-        '  $ wt cron schedule 5m ./sample-webtasks/hello-world.js',
-        '2. Create a webtask that runs every Tuesday at 09:05:',
-        '  $ wt cron schedule "5 9 * * 2" ./sample-webtasks/hello-world.js',
-    ].join('\n'),
+    description: '[DEPRECATED] Please use wt cron create or wt cron update',
```

Comment:
Maybe just put quotes around the cmds:  '[DEPRECATED] Please use \'wt cron create\' or \'wt cron update\''

Reply:
Fixed in 6a3a0ac.